moveit_ros/planning/planning_scene_monitor/include/moveit/planning_scene_monitor/planning_scene_monitor.h

```diff
@@ -462,13 +464,14 @@ class PlanningSceneMonitor : private boost::noncopyable
   planning_scene::PlanningSceneConstPtr scene_const_;
   planning_scene::PlanningScenePtr parent_scene_;  /// if diffs are monitored, this is the pointer to the parent scene
   boost::shared_mutex scene_update_mutex_;         /// mutex for stored scene
-  ros::Time last_update_time_;                     /// Last time the state was updated
-  ros::Time last_robot_motion_time_;               /// Last time the robot has moved
+  rclcpp::Time last_update_time_;                     /// Last time the state was updated
+  rclcpp::Time last_robot_motion_time_;               /// Last time the robot has moved
 
-  ros::NodeHandle nh_;
-  ros::NodeHandle root_nh_;
-  ros::CallbackQueue queue_;
-  std::shared_ptr<ros::AsyncSpinner> spinner_;
+  // ros::NodeHandle nh_;
```

Comment:
If we're not using these handles, maybe we can remove them?

Reply:
It was under the private: part, I switched it to protected as the nh_

https://github.com/ros-planning/moveit2/commit/5e641fde06500336ec31a933092439359628c41e